vayesta/solver/coupling.py

```diff
@@ -399,13 +533,21 @@ def externally_correct(solver, external_corrections):
     # CCSD uses exxdiv-uncorrected Fock matrix:
     fock = emb.get_fock(with_exxdiv=False)
 
+    if any([corr[1] == 'external-ccsdv' for corr in external_corrections]):
+        # At least one fragment is externally corrected, *and* contracted with
+        # integrals in the parent cluster. Form the required integrals
+        # for this parent cluster. Note that not all of these are needed.
+        fov_x, govov_x, gvvov_x, gooov_x, govoo_x = _integrals_for_extcorr(fx, fock)
+
     for y, corrtype, projectors in external_corrections:
 
         fy = frag_dir[y] # Get fragment y object from its index
         assert (y != fx.id)
 
-        if corrtype == 'external':
-            dt1y, dt2y = _get_delta_t_for_extcorr(fy, fock)
+        if corrtype == 'external' or 'external-fciv':
```

Comment:
This `or` is probably not what you wanted here 😅

Reply:
Indeed - thanks for the catch!